### PR TITLE
ARROW-3198: [Website] Blog post for the 0.11 release

### PR DESCRIPTION
**Pull request by wesm**
I think I got most of the highlights, but probably missed some things. Let me know any edits you want to make

### CODE REVIEW

File: site/_posts/2018-10-09-0.11.0-release.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow team is pleased to announce the 0.1.0 release. It is the
```
Comment (kszucs) [inline site/_posts/2018-10-09-0.11.0-release.md]: `0.11.0` pretty fatal typo

**Comment by wesm** [comment]
I will merge this tomorrow morning (CEST) and update the website so we can announce on social media, etc.

File: site/_posts/2018-10-09-0.11.0-release.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+In the coming months, we will continue to make progress on many fronts, with
+Gandiva packaging, expanded language support (especially in R), and improved
+data access (e.g. CSV, Parquet files) in focus.
+
+[1]: https://issues.apache.org/jira/issues/?jql=project%20%3D%20ARROW%20AND%20status%20in%20(Resolved%2C%20Closed)%20AND%20fixVersion%20%3D%200.11.0
+[2]: https://arrow.apache.org/install
+[3]: https://arrow.apache.org/release/0.11.0.html
+[4]: https://www.apache.org/dyn/closer.cgi/arrow/arrow-0.11.0/binaries
+[5]: http://dremio.com
```
Comment (kou) [inline site/_posts/2018-10-09-0.11.0-release.md]: https will be better.

File: site/_posts/2018-10-09-0.11.0-release.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+[6]: http://github.com/dremio/gandiva
```
Comment (kou) [inline site/_posts/2018-10-09-0.11.0-release.md]: ditto.

File: site/_posts/2018-10-09-0.11.0-release.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+[7]: https://github.com/kou
+[8]: https://github.com/apache/arrow/tree/master/r
+[9]: https://github.com/numba/numba
+[10]: http://gpuopenanalytics.com/
+[11]: http://grpc.io
```
Comment (kou) [inline site/_posts/2018-10-09-0.11.0-release.md]: ditto.

**Review by kou — APPROVED** [review]
+1

**Pull request closed without merging**